pkg/controller/registry/resolver/rbac.go

```diff
@@ -147,7 +147,7 @@ func RBACForClusterServiceVersion(csv *v1alpha1.ClusterServiceVersion) (map[stri
 		// Create ClusterRoleBinding
 		roleBinding := &rbacv1.ClusterRoleBinding{
 			ObjectMeta: metav1.ObjectMeta{
-				Name:      generateName(role.GetName(), role),
+				Name:      generateName(role.GetName(), permission),
```

Comment:
can we still get a collision if the same operator is installed into two different namespaces?

Reply:
Good point! Probably yes. I will look into this.
Would it make this bug more severe?

Reply:
I don't think it will affect the severity of the bug. Your current fix is still shrinking the number of possible collision cases, just not to zero. At a glance, I think we just need to add the CSV namespace to the `generateName` input somehow.